src/base/boundary_types.py

```diff
@@ -0,0 +1,33 @@
+"""Classes (really structs) for describing boundary layers.
+
+Types defined:
+  ShapefileFieldMapping: Mapping from canonical field names in the Django model
+    to the field names in a shapefile.
+  Boundary: Metadata about one boundary layer.
+  Boundaries: A list of one or more boundary layers.
+"""
+
+from typing import TypedDict
+from typing_extensions import NotRequired
+
+
+class ShapefileFieldMapping(TypedDict):
+    geometry: str
```

Comment:
can this be an enum?

Reply:
I'm not sure how.  Can you explain more?

Reply:
I was thinking something like this - https://medium.com/@bencleary/using-enums-as-django-model-choices-96c4cbb78b2e. It seems like there are only 10 or so geometry types (e.g. MULTIPOLYGON), so it could be nice to limit the acceptable choices.